pctiler/pctiler/endpoints/legend.py

```diff
@@ -23,7 +23,7 @@ async def get_classmap_legend(classmap_name: str) -> ORJSONResponse:
             status_code=404, detail=f"Classmap {classmap_name} not found"
         )
 
-    return ORJSONResponse(content=classmap)
+    return JSONResponse(content=classmap)
```

Comment:
ORJSONResponse will serialize faster than JSONResponse; is there a reason not to use ORJSONResponse?

Reply:
Sorry, the details for this are in the commit message. The ORJSON serializer requires that dict keys are strings, which the legend configs are not (ints/floats) because they're also used as the tiler colormap config. ORJSON itself can accept some options to check and convert key types, but the docs say this slows it down, and the fastapi class doesn't accept any options. I figured that the complexity of maintaining a custom response type that somewhat degrades the benefits of ORJSON wasn't worth simply using the default, at least for this particular endpoint. The test will prevent someone accidentally switching back to ORJSONResponse in the future.

Reply:
Ah, should have checked the commit msg - got it. That's a bit surprising and good to know for the future; sorry for the switch that caused the break!

Reply:
No problem - without the test, there was no way of knowing if it was working besides running the Explorer, so my bad on that.